src/components/ActionsBuilder/SupportedActions/SetPermissions/SetPermissionsEditor.tsx

```diff
@@ -313,6 +313,7 @@ const Permissions: React.FC<ActionEditorProps> = ({
                         activeTab === TABS.ASSET_TRANSFER ? 0 : field.value || 0
                       )}
                       disabled={activeTab === TABS.ASSET_TRANSFER}
+                      defaultValue={'0'}

```

Comment:
```suggestion
                      defaultValue={
                        activeTab === TABS.ASSET_TRANSFER ? null : '0'
                      }
```

I'd add this to prevent editing the amount field in the "Asset Transfer" tab.